### PR TITLE
Fix Erai-Raws indexer

### LINKED ISSUE #4116: [REQ] Erai-raws
Tracker Name : erai-raws
Type : Anime
~~Public~~ Semi-private  tracker
Indexer URL : https://erai-raws.info/
Indexer language : English , Spanish, Portuguese , French , Arabic , Russia

Thank You Very Much .
-------------------------------

### PR DESCRIPTION
**Pull request by 6cUbi57z**
Resolves #4116 by switching to new RSS feed

New feed supplies the size and quality information and has a link to the web page for the release in the description so this indexer has been cleaned up to remove quote a lot of the guesswork and some of the title parsing. I've still left in some of the title parsing as being able to clean up the season and episode numbers is very useful.

I've also done a minor styling fix for the settings page.

~~I added a common utility for converting sizing information figuring it may be useful for other indexers and could be expanded in future but I'm fine with it being moved into the indexer if preferred.~~ [edit] duplicate of existing util

### CODE REVIEW

File: src/Jackett.Common/Indexers/EraiRaws.cs [hunk 1]
```diff
@@ -16,25 +17,22 @@ namespace Jackett.Common.Indexers
...
+            // "https://erairaws.nocensor.space/"
         };
 
         public EraiRaws(IIndexerConfigurationService configService, Utils.Clients.WebClient wc, Logger l,
             IProtectionService ps, ICacheService cs)
             : base(id: "erai-raws",
                    name: "Erai-Raws",
                    description: "Erai-Raws is a team release site for Anime subtitles.",
-                   link: "https://www.erai-raws.info/",
+                   //link: "https://www.erai-raws.info/",
```
Comment (garfield69) [inline src/Jackett.Common/Indexers/EraiRaws.cs:34]: you can use the LegacySiteLinks override to automatically update users configs from old domain to new default domain
see for example
https://github.com/Jackett/Jackett/blob/8b59fb0a5281f280a50e34c7165897e43c36980f/src/Jackett.Common/Indexers/MoreThanTV.cs#L29-L31

**Review by garfield69 — CHANGES_REQUESTED** [review]

File: src/Jackett.Common/Indexers/EraiRaws.cs (hunk 1 above)
Comment (6cUbi57z) [reply src/Jackett.Common/Indexers/EraiRaws.cs:39]: Nice! Updated

File: src/Jackett.Common/Utils/SizeUtil.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
+using System.Text.RegularExpressions;
```
Comment (garfield69) [inline src/Jackett.Common/Utils/SizeUtil.cs:1]: I believe there is already a size utility see src/Jackett.Common/Models/ReleaseInfo.cs
for example
https://github.com/Jackett/Jackett/blob/483a72babd7cbf73d0ec9add8624e7feb054d3f3/src/Jackett.Common/Indexers/TVVault.cs#L132

Comment (6cUbi57z) [reply src/Jackett.Common/Utils/SizeUtil.cs]: Ah I was expecting it to be somewhere in utils if it existed. Converted over.

File: src/Jackett.Common/Content/custom.css [hunk 3]
```diff
@@ -51,6 +51,10 @@ body {
    width: 60px;
 }
 
+.setup-item-label:empty {
```
Comment (garfield69) [inline src/Jackett.Common/Content/custom.css:54]: please update src/Jackett.Common/Content/index.html in order to flush the browser cache to force loading new css
by updating the changed tag. suggestion:  PR12083
https://github.com/Jackett/Jackett/blob/7304ec222b8a3d25552c62ea9d8603e7f8c4a28e/src/Jackett.Common/Content/index.html#L31

**Review by garfield69 — APPROVED** [review]

**Comment by garfield69** [comment]
Thank you.

**Pull request merged**